pyphi/repertoire.py

```diff
@@ -52,7 +53,7 @@ def forward_cause_probability(
         mechanism=purview,
         purview=mechanism,
         mechanism_state=purview_state,
-        use_backward_tpm=True
+        direction=Direction.CAUSE
```

Comment:
This is nit-picky, but please make sure you have the `black` formatter set up to run on all your Python files in your IDE. It makes sure we have consistent formatting throughout the codebase. In this case, it will add a trailing comma here (all arguments on their own line get trailing commas, even the last ones).